libs/portals/admin/form-system/src/screens/Applications/Applications.tsx

```
@@ -20,15 +34,45 @@ export const Applications = () => {
       />
-      <div>{formatMessage(m.applications)}</div>
+      <Box marginTop={5}></Box>
+      <ApplicationsTableHeader />
+      {applicationsState &&
+        applicationsState?.map((a) => (
+          <ApplicationsTableRow
+            key={a?.id}
+            submittedAt={a?.submittedAt}
+            status={a?.status}
+          />
+        ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)